### PR TITLE
// Separate active filters from facets

### PR DESCRIPTION
**Pull request by maximebiloe**

### CODE REVIEW

File: themes/StarterTheme/templates/catalog/_partials/active_filters.tpl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{if $activeFilters|count}
+  <section class="active_filters">
+  <h1 class="h3">{l s='Active Filters'}</h1>
```
Comment (inetbiz) [inline themes/StarterTheme/templates/catalog/_partials/active_filters.tpl]: I'm concerned that not having `<aside><nav>~</nav></aside>` would be adding the H1 into the main body meta picked up in search. I've seen it before with left column infoboxes.

Comment (maximebiloe) [inline themes/StarterTheme/templates/catalog/_partials/active_filters.tpl]: Hi @inetbiz,

Would you mind making a PR with your suggestions, please ?

Regards

**Comment by inetbiz** [comment]
@maximebiloe checkout your commit or make a new PR?

**Comment by maximebiloe** [comment]
You can make a new PR directly on `feat/starter-theme` branch.
Thanks

**Pull request merged**